app/routes/_landing/_index/route.tsx

```diff
@@ -64,7 +64,7 @@ const LandingPage = () => {
                 <DisableTabIndex>
                   <Pod className="ml-52 h-min w-screen max-w-screen-2xl shrink-0 overflow-hidden" variant="alt">
                     <CharacterSheetProvider character={character}>
-                      <CoreTab />
+                      <CoreTab character="test" />
```

Comment:
🚫 **[tsc]** <2322> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Type 'string' is not assignable to type '{ id: string; name: string; race: string; level: number; class: string; background: string; alignment: Alignment; savingThrows: StatType[]; armorClass: number; speed: string; ... 5 more ...; userId: string; }'.